server/paymentProviders/stripe/webhook.ts

```diff
@@ -500,12 +568,31 @@ function formatPaymentMethodName(paymentMethod: Stripe.PaymentMethod) {
     case PAYMENT_METHOD_TYPE.SEPA_DEBIT: {
       return `${paymentMethod.sepa_debit.bank_code} ****${paymentMethod.sepa_debit.last4}`;
     }
+    case 'card': {
+      return paymentMethod.card.last4;
+    }
     default: {
       return '';
     }
   }
 }
 
+function mapStripePaymentMethodExtraData(pm: Stripe.PaymentMethod): object {
+  if (pm.type === 'card') {
+    return {
+      brand: pm.card.brand,
+      country: pm.card.country,
+      expYear: pm.card.exp_year,
+      expMonth: pm.card.exp_month,
+      funding: pm.card.funding,
+      fingerprint: pm.card.fingerprint,
```

Comment:
I think we `fingerprint` only exists when creating a new payment method with a token. We need to take care we're not overwriting the original token with an undefined or null value.

Reply:
This is one card created from the PaymentElement component. Not sure if it exists for all methods, but this one is only for cards.

https://dashboard.stripe.com/test/payments/pi_3MS3rLDjPFcHOcTm3QN0oedH

![image](https://user-images.githubusercontent.com/5448927/213551368-e0b2d6c5-861c-4afd-9799-fd4cf14a4e04.png)